npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -253,8 +254,16 @@ async function getAngularCliWebpackConfig (devServerConfig: AngularWebpackDevSer
   return config
 }
 
+function removeSourceMapPlugin (config: Configuration) {
+  config.plugins = config.plugins?.filter((plugin) => {
+    return (plugin as any).constructor.name !== 'SourceMapDevToolPlugin'
```

Comment:
I wonder if we can get types, we seem to have them here when we do something similar:

https://github.com/cypress-io/cypress/blob/develop/npm/webpack-dev-server/src/makeWebpackConfig.ts#L53

Alternatively, if we aren't sure of the type, maybe we should be defensive

```suggestion
    return (plugin as any)?.constructor?.name !== 'SourceMapDevToolPlugin'
```

Reply:
Seems like this cast isn't necessary after all, removing